File: python/1035.py

e = input().split(' ')
a = int(e[0])
b = int(e[1])
c = int(e[2])
d = int(e[3])

if b > c and d > a and (c+d) > (a+b) and c > 0 and d > 0 and a%2==0:
    print("Valores aceitos")
else:
    print("Valores nao aceitos")
    

File: php/1035.php
<?php
    $linha = fgets(STDIN);
    $linha = explode(' ', $linha);
    $a = $linha[0];
    $b = $linha[1];
    $c = $linha[2];
    $d = $linha[3];

    if ($b > $c && $d > $a && ($c+$d) > ($a+$b) && $c > 0 && $d > 0 && $a%2==0)
        printf("Valores aceitos\n");
    else
        printf("Valores nao aceitos\n");


?>